File: app/src/main/java/com/cylonid/nativealpha/model/WebAppInstanceCreator.java
package com.cylonid.nativealpha.model;

import com.google.gson.InstanceCreator;

import java.lang.reflect.Type;

public class WebAppInstanceCreator implements InstanceCreator<WebApp>
{
    @Override
    public WebApp createInstance(Type type)
    {
        return new WebApp("", Integer.MAX_VALUE);
    }
}

File: app/src/main/kotlin/com/cylonid/nativealpha/model/WebAppInstanceCreator.kt
package com.cylonid.nativealpha.model

import com.google.gson.InstanceCreator
import java.lang.reflect.Type

class WebAppInstanceCreator : InstanceCreator<WebApp> {
    override fun createInstance(type: Type): WebApp {
        return WebApp("", Int.MAX_VALUE)
    }
}